code/modules/mob/living/simple_animal/constructs.dm

```diff
@@ -45,6 +45,12 @@
 	mobsay_color = "#FF6262"
 	mobchatspan = "cultmobsay"
 
+/mob/living/simple_animal/hostile/construct/get_num_legs()
+	return 0
```

Comment:
If they have no legs it should return `FALSE`. It doesn't matter if it's boolean or not, byond treats 'FALSE' as 0 but it's easier to read.

Reply:
By that logic, it should return TRUE if the amount of legs is at whatever value the define for TRUE is?

No, that's bad code and will lead to confusion. It's not a boolean value, it's never treated as a boolean value, it's compared to as a number would, therefor it should never be written or suggested that it can hold a TRUE or FALSE value.

Reply:
There is no need to be hostile against me.
If it has legs it will return number of its legs else it would return `FALSE` (no legs). How does that sound for you?

Reply:
I'm not being hostile towards you. I'm trying to explain to you in as much detail as possible why returning a boolean value for what is a numeric amount is not good code.

When an entity has no legs, it has exactly 0 legs. It doesn't have FALSE legs, just as an entity with 1 legs doesn't have TRUE legs. Do you understand?

Reply:
ghost is correct. Boolean should only be used for true/false cases. They should almost never be used in conjunction with numerical values, it actually makes it _harder_ to read.